.github/workflows/snap.yml

```diff
@@ -0,0 +1,60 @@
+name: Snap
+
+on:
+  pull_request:
+    branches: [main]
+
+env:
+  SNAP_NAME: chisel
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    outputs:
+      chisel-snap: ${{ steps.build-chisel-snap.outputs.snap }}
+
+    steps:
+      - name: Checkout chisel repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Build chisel Snap
+        id: build-chisel-snap
+        uses: snapcore/action-build@v1
+
+      - name: Attach chisel snap to GH workflow execution
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ steps.build-chisel-snap.outputs.snap }}
+          path: ${{ steps.build-chisel-snap.outputs.snap }}
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    needs: [build]
+    outputs:
+      chisel-version: ${{ steps.install-chisel.outputs.version }}
+
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: ${{ needs.build.outputs.chisel-snap }}
+
+      - name: Install the chisel snap
+        id: install-chisel
+        run: |
+          set -ex
+          # Install the chisel snap from the artifact built in the previous job
+          sudo snap install --dangerous ${{ needs.build.outputs.chisel-snap }}
+
+          # Make sure chisel is installed
+          echo "version=$(chisel version)" >> "$GITHUB_OUTPUT"
```

Comment:
give the above comment, you won't need this either. printing the version should be enough

Reply:
Updated it in https://github.com/canonical/chisel/pull/92/commits/ca35b7fdd97bab09751cfbdb5280385874ec905e. Thanks!